lib/net.js

```diff
@@ -620,7 +608,7 @@ Socket.prototype.read = function(n) {
       !this._handle.reading) {
     tryReadStart(this);
   }
-  return ReflectApply(stream.Duplex.prototype.read, this, [n]);
+  return stream.Duplex.prototype.read.call(this, n);
```

Comment:
Do we prefer `.call` or `Reflect.apply`?

Reply:
We should prefer `.call` when the number of arguments is fixed. This avoids allocation of an array.